src/support/z_session_page.erl

```diff
@@ -216,6 +216,7 @@ add_script(Script, Context) ->
     z_transport:page(javascript, Script, Context).
 
 %% @doc Split the scripts from the context and add the scripts to the page.
+-spec add_script(#context{}) -> #context{}.
```

Comment:
The spec in line 219 uses a record, please define a type for the record and use that instead.